manifests/proxydns.pp

```diff
@@ -21,9 +21,29 @@
     soaip   => $ip,
   }
 
-  ::dns::zone { $foreman_proxy::dns_reverse:
-    soa     => $::fqdn,
-    reverse => true,
-    soaip   => $ip,
+  $foreman_proxy::dns_reverse.each |String $reverse| {
```

Comment:
This module currently supports Puppet 3 with the default parser, so this iteration syntax isn't valid.